.eslintrc.js

```diff
@@ -64,6 +64,10 @@ const baseTypeScriptRules = {
     "@typescript-eslint/quotes": "off",
     "@typescript-eslint/lines-between-class-members": ["error", "always", { exceptAfterSingleLine: true }],
     "@typescript-eslint/no-unnecessary-type-assertion": ["off"],
+    "import/prefer-default-export": "off",
```

Comment:
I think we are using eslint-disable a lot for this rule, so I'm thinking on disabling it. Thoughts?